apps/settings/js/modules/keyboard_context.js

```diff
@@ -189,15 +189,10 @@ define(function(require) {
         var keyboardAppInstance = keyboard.app;
         var keyboardManifest =
           new ManifestHelper(keyboardAppInstance.manifest);
-        var inputs = keyboardManifest.inputs;
         keyboard.name = keyboardManifest.name;
         keyboard.description = keyboardManifest.description;
         keyboard.layouts.forEach(function(layout) {
-          var key = layout.id;
-          var layoutInstance = inputs[key];
           layout.appName = keyboardManifest.name;
-          layout.name = layoutInstance.name;
-          layout.description = layoutInstance.description;
```

Comment:
We don't need to update name/desc of a layout in localized event because these names are not localized.